x-pack/test_serverless/api_integration/test_suites/observability/slos/delete_slo.ts

```diff
@@ -169,7 +170,7 @@ export default function ({ getService }: FtrProviderContext) {
             sloId,
             indexName: SLO_DESTINATION_INDEX_PATTERN,
           });
-          if (sloRollupDataAfterDeletion.hits.hits.length > 0) {
+          if (sloRollupDataAfterDeletion.hits.hits.length > numberOfRollupDocumentsBeforeDeletion) {
```

Comment:
Should we do `>=`. Wondering if there might be a false positive here otherwise where additional documents have not been added during the small testing window, but also none have been deleted.